js/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "forceConsistentCasingInFileNames": true
   },
   "include": [
-    "src/**/*.ts"
+    "src/**/*.ts",
+    "bench/*.ts"
```

Comment:
Is this necessary to get the suite to run? When building (`npm run build`) this nests the code in a way that dependent libraries get confused by (`dist/src/**` instead of `dist/**`). Can we either remove this (so the directories aren't nested) or update `package.json` to point to `main: dist/src/index.js` and `types: dist/src/index.d.ts`? Either way should work for me.

Reply:
Yes, will fix.

Reply:
fixed in 9b75872